migration/mysql/20241121081328_add_api_key_maintainer_table.sql

```diff
@@ -0,0 +1,2 @@
+-- Modify "api_key" table
+ALTER TABLE `api_key` ADD COLUMN `api_key` varchar(255) NULL, ADD COLUMN `maintainer` varchar(255) NULL;
```

Comment:
Instead of using NULL, how about setting NOT NULL and the default value to empty?
Once the migration is finished, we can remove the default value.
WDYT?

```suggestion
ALTER TABLE `api_key` ADD COLUMN `api_key` varchar(255) NOT NULL DEFAULT "", ADD COLUMN `maintainer` varchar(255) NOT NULL DEFAULT "";
```

Reply:
Yeah that can avoid some corner cases, thanks! and I have added description column